libs/tokens/src/hooks/tokens/useNetworkLogo.ts

```diff
@@ -0,0 +1,41 @@
+import { useMemo } from 'react'
+
+import { BaseChainInfo, getChainInfo } from '@cowprotocol/common-const'
+
+import { useBridgeSupportedNetworks } from '../../../../../apps/cowswap-frontend/src/modules/bridge/hooks/useBridgeSupportedNetworks'
```

Comment:
You cannot import files from apps into libs

Reply:
There is `TokensListsUpdater`, you can pass the value through it and store it in `environmentAtom`